lib/pf/UnifiedApi.pm

```diff
@@ -112,6 +112,22 @@ our @API_V1_ROUTES = (
             },
         },
     },
+    { 
+        controller => 'Services', 
+        resource => undef,
+        collection => {
+            http_methods => undef,
+            subroutes => {
+                ':service/status' => { get => 'status' },
+                ':service/start' => { post => 'start' },
+                ':service/stop' => { post => 'stop' },
+                ':service/restart' => { post => 'restart' },
+                ':service/enable' => { post => 'enable' },
+                ':service/disable' => { post => 'disable' },
+            }
+        } 
+    },
+
```

Comment:
This routing does not match our convention.

It should be the following instead.
GET    /services <- Get the list of services
GET    /service/:service_id <- This returns the status
POST /service/:service_id/start
POST /service/:service_id/stop
POST /service/:service_id/restart
POST /service/:service_id <- This should accept a parameter {status => 'enabled'}


The config for that is.

```perl
{
    controller => 'Services',
    resource   => {
        http_methods => {
            get  => 'status',
            post => 'update'
        },
        subroutes => {
            'start'   => { post => 'start' },
            'stop'    => { post => 'stop' },
            'restart' => { post => 'restart' },
            'enable'  => { post => 'enable' },
            'disable' => { post => 'disable' },
        },
    },
    collection => {
        http_methods => {
            get => 'list',
        }
    },
}
```